.github/workflows/tests.yml

```diff
@@ -68,3 +68,6 @@ jobs:
 
       - name: Acceptance Tests postgres
         run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -s acceptance -d postgres
+
+      - name: Build CSS
+        run: Build/Scripts/runTests.sh -s buildCss
```

Comment:
Should we combine this together with a "git repo" clean check afterwards ? Like in core. This would check a) that compiling is possible and b) compilation has been pushed along with changes ? (and no left over)

Reply:
currently only interested in "compiles at all".